<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsDialog</class>
 <widget class="QDialog" name="SettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>686</width>
    <height>457</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="closeButton">
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="displaySettings">
      <attribute name="title">
       <string>Display</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0" colspan="3">
        <widget class="QCheckBox" name="useWorkspaceRoundingCheckBox">
         <property name="text">
          <string>Use workspace rounding</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="roundingMinutesCaptionLabel">
         <property name="text">
          <string>Rounding Minutes</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1" colspan="2">
        <widget class="QComboBox" name="roundingMinutesComboBox">
         <item>
          <property name="text">
           <string>No Rounding</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>1 Minute</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>5 Minutes</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>6 Minutes</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>10 Minutes</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>12 Minutes</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>15 Minutes</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>30 Minutes</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>1 Hour</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>4 Hours</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="2" column="0" rowspan="2" colspan="2">
        <widget class="QLabel" name="roundingCaptionLabel">
         <property name="text">
          <string>Rounding</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1" colspan="2">
        <widget class="QComboBox" name="roundingComboBox">
         <item>
          <property name="text">
           <string>No Rounding</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Up</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Nearest</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Down</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="4" column="0" colspan="3">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>613</width>
           <height>222</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="exportSettings">
      <attribute name="title">
       <string>Export</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="exportDefaultLocationCaptionLabel">
         <property name="text">
          <string>Default Location</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QLineEdit" name="exportTitleEdit"/>
       </item>
       <item row="2" column="2">
        <widget class="QLineEdit" name="exportSubjectEdit"/>
       </item>
       <item row="0" column="2">
        <widget class="QLineEdit" name="exportDefaultLocationEdit"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="exportSubjectCaptionLabel">
         <property name="text">
          <string>Subject</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QPushButton" name="selectExportDefaultLocationButton">
         <property name="text">
          <string>Select ...</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="exportTitleCaptionLabel">
         <property name="text">
          <string>Title</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="exportCreatorCaptionLabel">
         <property name="text">
          <string>Creator</string>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="QLineEdit" name="exportCreatorEdit"/>
       </item>
       <item row="4" column="2">
        <widget class="QLineEdit" name="exportCompanyEdit"/>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="exportCompanyCaptionLabel">
         <property name="text">
          <string>Company</string>
         </property>
        </widget>
       </item>
       <item row="5" column="2">
        <widget class="QLineEdit" name="exportDescriptionEdit"/>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="exportDescriptionCaptionLabel">
         <property name="text">
          <string>Description</string>
         </property>
        </widget>
       </item>
       <item row="6" column="2">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
